syntax = "proto3";

package enhanced_protos;

option cc_generic_services = false;
option java_generic_services = false;
option py_generic_services = false;

// Entry represents a log entry's data with deduplication information.
message Entry {
  // The client ID for this log entry.
  string client_id = 1;
  // The request ID for this log entry.
  string request_id = 2;
  // The data contained in the log entry.
  bytes data = 3;
}